components/inputs/docs/input-checkbox.md

```diff
@@ -24,43 +72,49 @@ The `<d2l-input-checkbox>` element can be used to get a checkbox and optional vi
 | `not-tabbable` | Boolean | Sets `tabindex="-1"` on the checkbox |
 | `value` | String | Value of the input |
 
-**Accessibility:**
+### Events
+
+When the checkbox's state changes, it dispatches the `change` event:
+
+```javascript
+checkbox.addEventListener('change', (e) => {
+  console.log(checkbox.checked);
+});
+```
+<!-- docs: end hidden content -->
+
+### Accessibility Properties
 
 To make your usage of `d2l-input-checkbox` accessible, use the following property when applicable:
 
 | Attribute | Description |
-|--|--|
+|---|---|
 | `aria-label` | Use when text on checkbox does not provide enough context |
 | `description` | Use when label on input does not provide enough context. |
 
+### Usage
+
 **Methods:**
```

Comment:
Are we including bold titles like this within daylight pages? Maybe we remove the `Methods` title or switch the H3 above it to `Usage Methods`

Reply:
I believe we'd discussed having a separate "usage" or "how to use" section in components where necessary and so calling it "usage" would keep that consistent, and then here within that is methods. I'll confirm with Jeff though.